### PR TITLE
Update for xblock static file handling.

### PR DESCRIPTION
**Pull request by cahrens**
TNL-4710

@tobz and @efischer19 please review. After this is merged, I will bump the ORA release number and do a new version.

I verified on my devstack that this works. There's no point in creating a sandbox for this because sandboxes worked before.

### CODE REVIEW

File: scripts/run-pep8.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-MAX_PEP8_VIOLATIONS=113
+MAX_PEP8_VIOLATIONS=110
```
Comment (cahrens) [inline scripts/run-pep8.sh]: While running tests, I noticed that we only had 110 pep8 violations, so I lowered this.

**Comment by tobz** [comment]
👍

**Comment by efischer19** [comment]
LGTM, confirmed working on my devstack as well.

👍

**Comment by efischer19** [comment]
Let me know when you need me to run translations for the ORA release

**Comment by cahrens** [comment]
Thanks @efischer19, can you go ahead and run translations? I will add to it the version bump.

**Pull request merged**